server/@types/models/OffenceDetail.ts

```diff
@@ -1,14 +1,14 @@
 export interface OffenceDetail {
   acceptsResponsibility: boolean
-  acceptsResponsibilityDetail: string
+  acceptsResponsibilityDetail: string | null
```

Comment:
I think we can just make these properties optional strings (`acceptsResponsibilityDetail?: string`) rather than `string | null`, to keep it more in line with the typing of other fields that are optional in the API

Reply:
I did think that initially but in this case, the key **_will_** be present, just potentially `null`.  Where as `acceptsResponsibilityDetail?: string` for example could be `undefined` but not `null`.

Reply:
I'm now doubting myself as I've just used openapi-typescript to generate the type from the schema (I suppose we'll be doing this soon in the future anyway) and it's given me 
```
  acceptsResponsibility?: boolean
  acceptsResponsibilityDetail?: string
  contactTargeting?: boolean
  domesticViolence?: boolean
  motivationAndTriggers?: string
  numberOfOthersInvolved?: number
  offenceDetails?: string
  othersInvolvedDetail?: string
  patternOffending?: string
  peerGroupInfluences?: string
  raciallyMotivated?: boolean
  recognisesImpact?: boolean
  repeatVictimisation?: boolean
  revenge?: boolean
  stalking?: boolean
  victimWasStranger?: boolean
  ```

which is the opposite of what I was expecting 😄 .  Will go with that.

Reply:
Ah! Good to confirm that that is how it maps things. I assume we can have faith in openapi-typescript 😅